docs/manipulation.rst

```diff
@@ -93,7 +96,7 @@ The median based smoothing  is implemented using `scipy.signal.medfilt` and
 has a similar call structure to the convolution-based smoothing methods. This
 method applys the median filter across the flux.
 
-Note: This method is not flux conserving.
+Note: This method is not flux conserving and errors are not propagated.
```

Comment:
```suggestion
Note that this method is not flux conserving and errors are not propagated.
```

Or it could be
```suggestion

.. note::
    This method is not flux conserving and errors are not propagated.
    
```

Related question: are they not propoagated because we just haven't implemented it yet, or because it's ambiguous what the right way is to do the propogation?  If it's the latter we should probably say that here (i.e. it's a feature not a bug :wink:)